packages/lodestar/src/metrics/metrics/lodestar.ts

```diff
@@ -160,6 +160,13 @@ export function createLodestarMetrics(
       }),
     },
 
+    gossipPeer: {
+      score: register.gauge<"peer">({
+        name: "lodestar_gossip_peer_score_total",
+        help: "Gossip peer score",
+        labelNames: ["peer"],
+      }),
+    },
```

Comment:
The label peer has too much cardinality (too many different values) and will break our metrics. Refer to https://www.robustperception.io/cardinality-is-key as example on rationale.

What we want to know is an idea of the peer's score overall, not per peer specifically. You should use the `AvgMinMax` metric and dump an array of scores on each scrape